packages/web-components/src/components/masthead/cloud/cloud-masthead-composite.ts

```diff
@@ -0,0 +1,150 @@
+  }
+
+  /**
+   * The cookie name for determining user login status.
+   */
+  @property({ type: String, reflect: true })
+  loginStatusCookieName = 'com.ibm.cloud.iam.LoggedIn.manual';
+
+  render() {
+    const {
+      activateSearch,
+      authenticatedProfileItems,
+      currentSearchResults,
+      platform,
+      platformUrl,
+      inputTimeout,
+      mastheadAssistiveText,
+      menuBarAssistiveText,
+      menuButtonAssistiveTextActive,
+      menuButtonAssistiveTextInactive,
+      language,
+      openSearchDropdown,
+      searchPlaceholder,
+      selectedMenuItem,
+      unauthenticatedProfileItems,
+      l1Data,
+      _loadSearchResults: loadSearchResults,
+    } = this;
+    const loginStatus = this._getUserLoginStatus();
+    const profileItems = loginStatus.user === 'authenticated' ? authenticatedProfileItems : unauthenticatedProfileItems;
+    return html`
+      <dds-left-nav-overlay></dds-left-nav-overlay>
+      <dds-left-nav>
+        ${!platform
+          ? undefined
+          : html`
+              <dds-left-nav-name href="${ifNonNull(platformUrl)}">${platform}</dds-left-nav-name>
+            `}
+        ${l1Data ? undefined : this._renderNavItems({ selectedMenuItem, target: NAV_ITEMS_RENDER_TARGET.LEFT_NAV })}
+        ${l1Data ? this._renderL1Items({ selectedMenuItem, target: NAV_ITEMS_RENDER_TARGET.LEFT_NAV }) : undefined}
+      </dds-left-nav>
+      <dds-masthead aria-label="${ifNonNull(mastheadAssistiveText)}">
+        <dds-masthead-menu-button
+          button-label-active="${ifNonNull(menuButtonAssistiveTextActive)}"
+          button-label-inactive="${ifNonNull(menuButtonAssistiveTextInactive)}"
+        >
+        </dds-masthead-menu-button>
+
+        ${this._renderLogo()}
+        ${!platform
+          ? undefined
+          : html`
+              <dds-top-nav-name href="${ifNonNull(platformUrl)}">${platform}</dds-top-nav-name>
+            `}
+        ${l1Data
+          ? undefined
+          : html`
+              <dds-top-nav menu-bar-label="${ifNonNull(menuBarAssistiveText)}">
+                ${this._renderNavItems({ selectedMenuItem, target: NAV_ITEMS_RENDER_TARGET.TOP_NAV })}
+              </dds-top-nav>
+            `}
+        <dds-masthead-search-composite
+          ?active="${activateSearch}"
+          input-timeout="${inputTimeout}"
+          language="${ifNonNull(language)}"
+          ?open="${openSearchDropdown}"
+          placeholder="${ifNonNull(searchPlaceholder)}"
+          .currentSearchResults="${ifNonNull(currentSearchResults)}"
+          ._loadSearchResults="${ifNonNull(loadSearchResults)}"
+        ></dds-masthead-search-composite>
+        <dds-masthead-global-bar>
+          <dds-cloud-masthead-profile ?authenticated="${loginStatus.user === 'authenticated'}">
+            ${profileItems?.map(
+              ({ title, url }) =>
+                html`
+                  <dds-masthead-profile-item href="${ifNonNull(url)}">${title}</dds-masthead-profile-item>
```

Comment:
As part of the new Cloud masthead work, we need to expose a CTA button component that links out to the console. This is a new feature that we'll need to account for within the authenticated state of the `dds-cloud-masthead-profile` component (see specs - https://ibm.ent.box.com/file/786859527229). 

![component--cloud-masthead-l0--design-specs--1_0_0_pdf___Powered_by_Box](https://user-images.githubusercontent.com/1815714/112381171-a7196180-8cc0-11eb-8cf2-7818c1dc7651.jpg)

As for the data/translation, we'll need to add this to our JSON file as well. @annawen1 @jeffchew Let me know if there are fields within the existing data structure that we can reuse or if this is something new that we need to add. https://1.www.s81c.com/common/carbon-for-ibm-dotcom/translations/cloud-masthead/jsononly/usen.json

Reply:
hmm don't see any existing fields we can utilize for the `console` button so I think we need to add it. Possibly in 

```
"masthead": {
    "search": {
      "btnSearchClosed": "Open IBM search field",
      "btnSearchOpen": "Search all of IBM",
      "btnClose": "Close IBM search field",
      "placeHolderText": "Search all of IBM"
    },
    --> "not sure key name": "Console"
```

Reply:
@annawen1 Looking at the existing structure, I think it might be good to add this button within the `[profileMenu][signedin]` array, as this button is intended to show up in the authenticated state (see attached ). This will be managed outside of the `[profileMenu][signedin][links]`, which is being used to display the user drop-down menu. Let me know what you think.

![https___1_www_s81c_com_common_carbon-for-ibm-dotcom_translations_cloud-masthead_jsononly_usen_json](https://user-images.githubusercontent.com/1815714/112498551-02992d00-8d5d-11eb-8a32-2ebda166ec35.jpg)

@huntermacd let me know what you think as well.

Reply:
yep, that looks good! Just note we need to edit the type files in the services store package! Let me know if you need help with that.

Reply:
@annawen1 Great! I'll work on opening a PR to add this new `ctaButton` array. As for accounting for these types in the services store package, perhaps this is something you can help us add once the data is in place?

Reply:
sure!

Reply:
Here's the PR to adjust the data structure based on our conversation here - https://github.ibm.com/webstandards/ibm-dotcom-translations/pull/295.